src/controllers/tokens/get.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response } from 'express';
+import TokenService from '@/services/TokenService';
+
+export const GetTokens = async (_req: Request, res: Response) => {
+    let status = 500;
+
+    const response = await TokenService.getAllTokens();
+
+    if (!response) {
+        res.status(status).send('No token found!');
```

Comment:
Response will never be undefined and thus this check becomes redundant.
When the collection is empty it'll return an empty object.
This can be scrapped.